pkg/cliClient/cliClient.go

```diff
@@ -77,3 +83,18 @@ func (c *CliClient) RequestEvaluation(request EvaluationRequest) (EvaluationResp
 
 	return *evaluationResponse, nil
 }
+
+func (c *CliClient) GetVersionMessage(ch chan VersionMessage, cliVersion string) {
+	defer close(ch)
```

Comment:
I would keep `GetVersionMessage` primitive => call endpoint and returns an answer
the channel is more of a BL plus rule of thumb to remember - the one that creates a channel should also close it